backtesting/lib.py

```diff
@@ -452,6 +452,43 @@ def next(self):
                                self.data.Close[index] + self.__atr[index] * self.__n_atr)
 
 
+class PercentageTrailingStrategy(Strategy):
+    """
+    A strategy with automatic trailing stop-loss, trailing the current
+    price at distance of some percentage. Call
+    `PercentageTrailingStrategy.set_trailing_sl()` to set said percentage
+    (`5` by default). See [tutorials] for usage examples.
+
+    [tutorials]: index.html#tutorials
+
+    Remember to call `super().init()` and `super().next()` in your
+    overridden methods.
+    """
+    _sl_pct = 5.
+
+    def init(self):
+        super().init()
+
+    def set_trailing_sl(self, percentage: float = 5):
+        assert percentage > 0, "percentage must be greater than 0"
```

Comment:
Everywhere else, the framework uses "pct" and "percent" to really mean [rate](https://en.wikipedia.org/wiki/Rate_(mathematics)) (e.g. 5% == 0.05). I wonder if here shouldn't be the same.

Reply:
So are you saying that a user should be inputting 0.05 instead of 5 and the code should use that value as is?